src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -169,14 +237,27 @@ export class SearchResults extends Component {
           </span>
           {resultsEndNumber} of {totalResults} results
         </h2>
+        {showYellowRibbonEnhancements && (
+          <AlertBox
+            content={TOOL_TIP_CONTENT}
+            headline={TOOL_TIP_LABEL}
+            isVisible={state.isSearchResultsTipOpen}
+            onCloseAlert={toggleAlertBoxAppearance(false)}
+            status="info"
+          />
```

Comment:
It's kind of confusing to use two different terms for this ('alertBox' as well as 'toolTip'). I think you could get away with just inlining the function like this and then removing `toggleAlertBoxAppearance` -

```
onCloseAlert={() => this.setState({ isSearchResultsTipOpen: !this.state.isSearchResultsTipOpen });
```

I might also consider renaming that to be `isSearchResultsToolTipOpen`. Sounds kinda funny reading it without the "tool" part in there lol.